erpnext/manufacturing/doctype/production_planning_tool/production_planning_tool.py

```diff
@@ -131,101 +154,155 @@ def get_items(self):
 					or item.is_sub_contracted_item = 1)) %s""" % \
 			(", ".join(["%s"] * len(so_list)), item_condition), tuple(so_list), as_dict=1)
 
-		return items + packed_items
+		self.add_items(items + packed_items)
+	
+	def get_mr_items(self):
+		mr_list = filter(None, [d.material_request for d in self.get('material_requests')])
```

Comment:
mr_list = [d.material_request for d in self.get('material_requests') if d.material_request]

It will also filter blank